### PR TITLE
Add identity and access control docs

### PR DESCRIPTION
**Pull request by jraddaoui**
Create an administrator section in the docs and add an initial identity and access control section.

Refs #957.

### CODE REVIEW

File: docs/src/admin-manual/iac.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+VITE_OIDC_ABAC_ENABLED
+VITE_OIDC_ABAC_CLAIM_PATH
+VITE_OIDC_ABAC_CLAIM_PATH_SEPARATOR
+VITE_OIDC_ABAC_CLAIM_VALUE_PREFIX
+```
+
+They must match the ones configured in the API. `VITE_OIDC_AUTHORITY` has to be
+the same OIDC provider URL and `VITE_OIDC_CLIENT_ID` needs to be the same or a
+trusted client. This client (and the one used in the API configuration, if they
+are not the same) must be included in the `aud` claim from the access token,
```
Comment (fiver-watson) [inline docs/src/admin-manual/iac.md]: ends w a comment instead of a period

File: docs/src/admin-manual/iac.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+The authorization flow will request the `openid email profile` scopes by
+default. If needed, `VITE_OIDC_EXTRA_SCOPES` can be used to request additional
+scopes. The ABAC variables will work in the same way as they do in the API,
+they are explained in detail in the API configuration comments above.
+
+These environment variables must be set at build time or they can be replaced
```
Comment (fiver-watson) [inline docs/src/admin-manual/iac.md]: you say "must", and then follow with an "or" - is the or a bad outcome if you don't do the _must_? Or is it an alternative? If it's an alternative, then I would change the "must" to "can" - _must_ is typically used as an imperative (you must do this, or else...) 

> These environment variables can be set at build time, or they can be replaced in the final assets.

**Review by fiver-watson — APPROVED** [review]
Overall, looks good - 2 minor grammatical suggestions

**Comment by jraddaoui** [comment]
Thanks @fiver-watson!

@djjuhasz @sbreker, I'll merge so it gets deployed to the docs site. I'll still appreciate your feedback when you have some time, will address it in a follow-up PR.

**Pull request merged**